lib/cli/cli.js

```diff
@@ -151,20 +151,22 @@ class CLI {
 
       let platformCheckerToken = heimdall.start('platform-checker');
 
+      const emberCLIVersion = require('../../package.json').version;
       const PlatformChecker = require('../utilities/platform-checker');
       let platform = new PlatformChecker(process.version);
       let recommendation =
-        ' We recommend that you use the most-recent "Active LTS" version of Node.js.' +
-        ' See https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md for details.';
+        ' See "https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md" to find out which version of Node is best to use.';
```

Comment:
Why did you wrap the link in double quotes? Can you still cmd-click (or equivalent) it in terminals?

Reply:
> Why did you wrap the link in double quotes?

Looked more highlighted to me, but I can revert.

> Can you still cmd-click (or equivalent) it in terminals?

Yes.

I can link to the CI file, the only downside is that code changes might affect the link and that it needs to be updated again (if we use line numbers).

Another (but probably far-fetched) solution could be to define `[12.x, 14.x, 16.x]` in a separate file we can link to and access that file in CI using [fromJson](https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson).

Reply:
We can also include the node versions directly in the message? We already load the CI file here:
https://github.com/ember-cli/ember-cli/blob/master/lib/utilities/platform-checker.js#L7.

Reply:
@locks Any suggestions on how I should update this message?